mlflow/tracking/client.py

```diff
@@ -875,6 +875,30 @@ def link_traces_to_run(self, trace_ids: list[str], run_id: str) -> None:
         """
         return self._tracking_client.link_traces_to_run(trace_ids, run_id)
 
+    def unlink_traces_from_run(self, trace_ids: list[str], run_id: str) -> None:
+        """
+        Unlink multiple traces from a run by removing entity associations.
+
+        Args:
+            trace_ids: List of trace IDs to unlink from the run.
+            run_id: ID of the run to unlink traces from.
+
+        Example:
+            .. code-block:: python
+
+                import mlflow
+                from mlflow import MlflowClient
+
+                client = MlflowClient()
+
+                # Unlink multiple traces from a run
+                client.unlink_traces_from_run(
+                    trace_ids=["trace_123", "trace_456", "trace_789"],
+                    run_id="run_abc",
+                )
+        """
+        return self._tracking_client.unlink_traces_from_run(trace_ids, run_id)
```

Comment:
Fixed in 30f30bcb74 - added `unlink_traces_from_run` to `TrackingServiceClient` following the same pattern as `link_traces_to_run`.